build.gradle

```diff
@@ -883,22 +944,22 @@ def sourceContent = project.copySpec {
   into kiekerPackagenamebase + "-" + kiekerVersion
 }
 
-task distributeSourceZip(type: Zip, group: 'distribution', dependsOn: [mainJar, emfJar, aspectJJar,
-  'kieker-examples:buildStandaloneExamples']) {
-  description = 'Distributes source archive.'
-  archiveClassifier = 'sources'
-  with sourceContent
-}
-
-task distributeSourceTar(type: Tar, group: 'distribution', dependsOn: [mainJar, emfJar, aspectJJar,
-  'kieker-examples:buildStandaloneExamples']) {
-  description = 'Distributes source archive.'
-  archiveClassifier = 'sources'
-  with sourceContent
-
-  compression = Compression.GZIP
-  archiveExtension = 'tar.gz'
-}
+//task distributeSourceZip(type: Zip, group: 'distribution', dependsOn: [mainJar, emfJar, aspectJJar,
```

Comment:
Why is this commented code left in the source?

Reply:
As we did not finally discuss to never ever support a zip based binary distribution.